model/LtiResultAliasStorage.php

```diff
@@ -64,15 +64,13 @@ public function storeResultAlias($deliveryExecutionId, $resultId)
             self::DELIVERY_EXECUTION_ID => $deliveryExecutionId,
             self::RESULT_ID => $resultId,
         ];
-
         $queryBuilder = $this->getQueryBuilder();
         $queryBuilder->delete($this->getTableName());
-        $queryBuilder->where(self::RESULT_ID . '=? OR ' . self::DELIVERY_EXECUTION_ID . '= ?');
-        $queryBuilder->setParameters([$resultId, $deliveryExecutionId]);
-        $this->persistence->query($queryBuilder->getSQL())->execute($queryBuilder->getParameters());
-
-        $result = $this->getPersistence()->insert(self::TABLE_NAME, $data) === 1;
-        return $result;
+        $queryBuilder->where(self::RESULT_ID . "=:" . self::RESULT_ID);
+        $queryBuilder->orWhere(self::DELIVERY_EXECUTION_ID . "=:" . self::DELIVERY_EXECUTION_ID);
+        $queryBuilder->setParameters([self::RESULT_ID => $resultId, self::DELIVERY_EXECUTION_ID => $deliveryExecutionId]);
+        $queryBuilder->execute();
```

Comment:
@bartlomiejmarszal I feel you've lost here that change https://github.com/oat-sa/extension-tao-ltideliveryprovider/pull/235/commits/ab9e8b238db80edca4a75ba920a02df648815788#diff-587fbe7a3518fdcaaea9478ada64d9cbR72
Was that on purpose?